pom.xml

```diff
@@ -50,45 +50,47 @@
 
     <!-- override to set exclusions per-project -->
     <errorprone.args />
-    <errorprone.version>2.30.0</errorprone.version>
+    <!-- NOTE: Do not update as this is the last version to support Java 11 -->
```

Comment:
@anuraaga @reta @CodePrometheus if any of you want to suggest a way to handle this, maybe maven profile? then we could have latest here and override with a maven profile to drop down when Java 11. Just there may be an ordering concern with the errorprone profile (e.g. if we can get the version drop profile to apply before the errorprone one that uses it)

Reply:
Actually, the simplest is to just not run errorprone on 11. In CI, we test with two JDKs, and this only needs to run once!